src/services/kratos/index.ts

```diff
@@ -54,3 +58,20 @@ export const validateKratosCookie = async (
     session,
   }
 }
+
+export const revokeKratosToken = async (token: string): Promise<void | KratosError> => {
```

Comment:
this is not used in this PR for info. we could have another PR and provide explicit test for it (it may be buggy on the kratos side so some tests would be actually useful)